srcpkgs/the-way/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'the-way'
+pkgname=the-way
+version=0.14.4
+revision=1
+build_style=cargo
+short_desc="Code snippets manager for your terminal"
+maintainer="jgart <jgart@dismail.de>"
+license="MIT"
+homepage="https://github.com/out-of-cheese-error/the-way"
+distfiles="https://github.com/out-of-cheese-error/${pkgname}/archive/refs/tags/v${version}.tar.gz"
+checksum=@7ffce9b744ccbc9cdf8c7968f50ce8b580d66d9d40c477d090b6a45aa5560ccd
```

Comment:
why are you using content checksum?

Reply:
Not sure, what should I be using instead?

Reply:
I just pushed the non content checksum so I'm resolving this conversation.